src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1946,5 +1947,67 @@ public String getVariableNamePattern() {
 					}
 				})
 				.serializer(new EnumSerializer<>(Attribute.class)));
+		EnumUtils<Horse.Color> horseColorUtils = new EnumUtils<>(Horse.Color.class, "horse colors");
+		Classes.registerClass(new ClassInfo<>(Horse.Color.class, "horsecolor")
+				.user("horse ?colou?rs?")
+				.name("Horse Color")
+				.description("Represents a horse's color.")
+				.usage(horseColorUtils.getAllNames())
+				.since("INSERT VERSION")
+				.parser(new Parser<Horse.Color>() {
+					
```

Comment:
Empty line BEGONE 😆 
```suggestion
```

Reply:
Yeah, my bad again, anyways thanks for doing this, I know it's painful to have to keep saying these things

Reply:
I thought methods had to have newlines before them though

Reply:
I mean, it's done this way in some spots. This is mainly a nitpick of mine, so whether you want to do it is up to you.

Also, no worries whatsoever! We appreciate the PRs 😃